### PR TITLE
Add hibernate and implement data models

### PR DESCRIPTION
**Pull request by 1jeanpaul1**
## Problem
Don't have data models, because we don't have hibernate

## Solution
Add hibernate dependency, implement data models, and expose endpoint for verification

## Testing
1. Pull down this branch
2. **Run and Build:** `docker-compose up --build`
3. Go to the following endpoint and confirm that they are valid:
http://localhost:8082/forum/post/v1/all (Will contain post data populated in `DataPopulation.java`
4. Go to http://localhost:9000/ and confirm the `Post` and `Comment`

### CODE REVIEW

**Comment by Mikerad1** [comment]
@1jeanpaul1 the service is failing. keep getting: 
image
and 
image

Not sure what you added that is causing this. will try do some research on the errors unless you know what is causing them?

File: docker-compose.yml [hunk 1]
```diff
@@ -34,5 +32,5 @@ services:
 volumes:
   dbdata:
   pgadmindata:
-networks:
```
Comment (Mikerad1) [inline docker-compose.yml]: Dont you think it is necessarry for them to be on a network?

**Comment by Mikerad1** [comment]
Works

**Comment by Mikerad1** [comment]
it doesn't create a new db, it just uses the default created one. that what you want?

**Review by fcgl — APPROVED** [review]
Tested and works

**Pull request merged**